src/corona_hakab_model/agent.py

```diff
@@ -59,21 +59,27 @@ def get_infection_ratio(self):
 
 
 class Circle:
-    __slots__ = "kind", "agent_count"
+    __slots__ = "kind", "agent_count", "added_total", "removed_total"
```

Comment:
tracking circle has ever_visited, which is the same as added total. removed total is just ever_visited - agent_count

Reply:
Missed `ever_visited`, and you're right about the `removed_total`. I'll change it